frontend/views/pages/DesignSystem.vue

```diff
@@ -1061,8 +1061,8 @@ page(
             td
               users-selector(
                 label='Search for users'
-                :usernames='form.searchUsers'
-                defaultValue='alexjin'
+                :users='form.searchUsers'
```

Comment:
If `:users` is supposed to take `memberIDs`, can you modify it take be specific (since "users" is unclear whether it's usernames or memberIDs)?